salt/cloud/clouds/gce.py

```diff
@@ -400,6 +400,24 @@ def __get_tags(vm_):
     return tags
 
 
+def __get_labels(vm_):
+    '''
+    Get configured labels.
+    '''
+    lbl = config.get_cloud_config_value(
+        'labels', vm_, __opts__,
+        default='[]', search_global=False)
+    # Consider warning the user that the labels in the cloud profile
+    # could not be interpreted, bad formatting?
+    try:
+        labels = literal_eval(lbl)
+    except Exception:  # pylint: disable=W0703
```

Comment:
Here you should be catching the exact exception that occurs, otherwise we may want to handle different exceptions with different cases

Reply:
This is the exact same block that is used elsewhere for all the other gce functions, i would leave it as is.

Reply:
Is this the rule of thumb here - I am asking so I know what is considered good or bad practice for the salt code base.

Reply:
The rule of thumb is usually be consistent in that module.

Good luck trying to get stuff to be consistent on the whole code base.  If we feel like this is wrong, then we should write a pylint plugin for it.

In reality, we don't really care what the lower exception is, because we just know it was unable to get labels for that vm.  It could be any one of a bunch of them, better to just catch them and throw it away and return that there were no labels.

Reply:
Gotcha, thanks for giving some insight